bentoml/_internal/bento/bento.py

```diff
@@ -138,14 +140,12 @@ def create(
         bento_fs = fs.open_fs(f"temp://bentoml_bento_{svc.name}")
         ctx_fs = fs.open_fs(build_ctx)
 
-        assert build_config.additional_models is not None
-
         model_tags = build_config.additional_models
         # Add Runner required models to models list
         for runner in svc.runners.values():
             model_tags += runner.required_models
 
-        models: t.List[Model] = []
+        models: "t.List[Model]" = []
```

Comment:
Why do we have to do this instead of 
```
t.List["Model"]
```?

Reply:
I prefer fully quoting the type stylistically. I suspect that unquoted type hints have a startup performance impact and are in general more annoying to use, so at some point I think it would be nice to start quoting all annotations.

That, or we could use future annotations but sadly that's 3.7+.